smithy-codegen-core/src/test/java/software/amazon/smithy/codegen/core/ArtifactMetadataTest.java

```diff
@@ -0,0 +1,42 @@
+package software.amazon.smithy.codegen.core;
+
+import org.junit.jupiter.api.Test;
+import software.amazon.smithy.model.node.ObjectNode;
+
+class ArtifactMetadataTest {
+
+    @Test
+    void toNode() {
```

Comment:
Your test names should be clearer about what they are testing. Use keywords like "asserts", "validates", "does", "doesNot", "throws", and "expects".